layout-management-webapps/src/main/webapp/vue-app/site-navigation/components/SiteNavigationAddNodeDrawer.vue

```diff
@@ -240,13 +269,14 @@ export default {
       this.visible = true;
       this.isScheduled = false;
       this.nodeType = 'Group';
-      this.disabled = true;
+      this.disableNodeId = false;
+      this.editMode= false;
       this.$refs.siteNavigationAddNodeDrawer.close();
     },
     createNode() {
```

Comment:
I thnink that we should rename it saveNode since it is dedicated for create and edit